### ISSUE TITLE
Mustang 2.15.0 Action PDF generates non-compliant PDF output when XML file contains attachments

### ISSUE THREAD

**Issue by msccip** [ZUGFeRD/mustangproject#584]
When the XML file contains attachments, the underlying KoSIT visualization generates JavaScript code that is transferred to the output PDF. This results in a non-compliant PDF/A-3b file:

Validating file "factur-x.pdf" for conformance level pdfa-3b
The key S has a value JavaScript which is prohibited.
The document does not conform to the requested standard.
The document contains actions types other than for navigation (Launch, JavaScript, ResetForm, etc.).
The document does not conform to the PDF/A-3b standard.

**Comment by jstaerk** [comment]
Ah I see and you cant complain upstream because the kosit switched issues off for https://github.com/itplr-kosit/xrechnung-visualization ;-)

**Comment by msccip** [comment]
Yes, unfortunately that is the case, but the original KoSIT visualization does not create PDF/A. Only "your" Mustang PDF action does that. But this generated PDF is not PDF/A compatible because links and JavaScript are not allowed. So I think it is a Mustang topic.
